### PR TITLE
update readme during release

### PR DESCRIPTION
**Pull request by metaclips**
Update README.md file during release

### CODE REVIEW

**Comment by etorreborre** [comment]
@metaclips can you integrate the `gradle update_readmes` task as a release step instead?

**Comment by metaclips** [comment]
> @metaclips can you integrate the `gradle update_readmes` task as a release step instead?

Ok. I'll do that

File: tools/docker/builder/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,23 @@
 FROM gcc:11.2.0@sha256:0eaeaee298c154ecd383cabe5301a5df9fde0cd3a729124caf574500b936389b
 
 ARG RUSTUP_INIT_VERSION=1.25.2
-ARG RUST_VERSION=1.68.0
+ARG RUST_VERSION=1.69.0
```
Comment (shanesveller) [inline tools/docker/builder/Dockerfile]: Can you please leave this out if possible, or else mention it more prominently in the PR title if we need to include it? This is kind of off-topic for the concern being tackled here.

From the project-level perspective, ideally we should be using a [Rustup toolchain file](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) in the repo root as a source of truth, and flowing to other areas from that. I believe this Dockerfile would already properly pick that up with minor changes.

File: tools/docker/builder/Dockerfile [hunk 2]
```diff
@@ -135,6 +139,15 @@ RUN set -xe; \
     echo "${JQ_SHA256}  jq-${JQ_OS}" | sha256sum -c; \
     mv jq-${JQ_OS} /usr/bin/jq; \
     chmod +x /usr/bin/jq; \
+# Install for release
+    cargo install --locked git-cliff --version 0.7.0; \
+    cargo install --locked tomlq --version 0.1.0; \
+    cargo install --locked cargo-release --version 0.21.0; \
```
Comment (shanesveller) [inline tools/docker/builder/Dockerfile:145]: Thank you for pinning the versions 👍🏻 

 I'm familiar with git-cliff already. Is `tomlq` a runtime dependency of `cargo-readme` in some way? Or already included in other release steps?

Comment (metaclips) [reply tools/docker/builder/Dockerfile:145]: tomlq is used in other release steps, we use it to get version of crates during release.

File: tools/docker/builder/Dockerfile (hunk 1 above)
Comment (metaclips) [reply tools/docker/builder/Dockerfile]: > This is kind of off-topic for the concern being tackled here.

Removed

**Review by etorreborre — APPROVED** [review]

**Pull request merged**